pkg/skaffold/util/util.go

```diff
@@ -107,6 +107,13 @@ func ExpandPathsGlob(workingDir string, paths []string) ([]string, error) {
 	return ret, nil
 }
 
+// HasMeta reports whether path contains any of the magic characters
+// recognized by filepath.Match.
+// This is a copy of filepath/match.go's hasMeta
+func HasMeta(path string) bool {
+	return strings.ContainsAny(path, "*?[")
```

Comment:
why does `[` have to be there? and if yes, why not `]` as well?

Reply:
I see - this is the copy of hasMeta from filepath/match.go - in that file there is one more meta character `^` for negation...but this is really just nitpicking :)